docs/toc.md

```diff
@@ -235,6 +235,7 @@
 ### [Working with LINQ](csharp/tutorials/working-with-linq.md)
 ### [Microservices hosted in Docker](csharp/tutorials/microservices.md)
 ### [Using String Interpolation](csharp/tutorials/string-interpolation.md)
+### [Using Attributes](csharp/tutorials/attributes.md)
```

Comment:
I bet this is the conflict. Using the sub TOCs, this would be in the docs/csharp/tutorials directory.